.github/workflows/ci-core-reusable.yml

```diff
@@ -216,13 +216,11 @@ jobs:
       - name: Fee projection tests
         run: ci_run zk test i fees
 
-      - name: Run revert test - DISABLED (EVM-539)
```

Comment:
there are two places in CI where it is disabled, you've turned on only one of those

Reply:
Ah ok thanks!